### PR TITLE
First tests for retro

### PR DESCRIPTION
**Pull request by SivanovaPro**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -105,6 +105,21 @@
             <artifactId>allure-maven-plugin</artifactId>
             <version>2.6</version>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
```
Comment (mikementor) [inline pom.xml:109]: what is it for?

File: docker/lanehealth-retro-local.yaml [hunk 2]
```diff
@@ -0,0 +1,87 @@
+version: '3'
```
Comment (mikementor) [inline docker/lanehealth-retro-local.yaml:1]: add line in readme for that part

**Review by mikementor — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (SivanovaPro) [reply pom.xml:109]: I deleted it. Traces of the past

File: docker/lanehealth-retro-local.yaml (hunk 2 above)
Comment (SivanovaPro) [reply docker/lanehealth-retro-local.yaml:1]: I added an explanation in readme

**Review by mikementor — APPROVED** [review]

**Pull request merged**